public/app/plugins/panel/graph2/GraphPanel.tsx

```diff
@@ -19,12 +22,29 @@ export class GraphPanel extends PureComponent<Props> {
     const { data, timeRange, width, height } = this.props;
     const { showLines, showBars, showPoints } = this.props.options;
 
-    let vmSeries: TimeSeriesVMs;
-    if (data) {
-      vmSeries = processTimeSeries({
-        data,
-        nullValueMode: NullValueMode.Ignore,
-      });
+    const vmSeries: TimeSeriesVMs = [];
+    for (let t = 0; t < data.length; t++) {
+      const table = guessColumnTypes(data[t]);
+      const timeColumn = getFirstTimeColumn(table);
+      if (timeColumn >= 0) {
```

Comment:
nitpick: invert if condition, ie if (timeColumn <= 0) continue ... to reduce nesting